src/main/java/net/consensys/orion/cmd/Orion.java

```diff
@@ -514,7 +532,7 @@ private void writePortsToFile(final Config config, final int nodePort, final int
     try (final FileOutputStream fileOutputStream = new FileOutputStream(portsFile)) {
       properties.store(
           fileOutputStream,
-          "This file contains the ports used by the running instance of Pantheon. This file will be deleted after the node is shutdown.");
+          "This file contains the ports used by the running instance of Besu. This file will be deleted after the node is shutdown.");
```

Comment:
This was probably an error when copying this logic from Pantheon in the first place. I believe the ports file is used to write what ports the process (in this case Orion) is using. This is useful when we have dynamically allocated ports.

This message should probably read like: 
```
This file contains the ports used by the running instance of Orion. This file will be deleted after the node is shutdown.
```

Reply:
Fixed!